src/js/base/stories/RelativeTime.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { RelativeTime } from "../RelativeTime";
+
+const currentDate = new Date();
+
+export default {
+    title: "base/RelativeTime",
+    component: RelativeTime
+};
+
+const Template = args => <RelativeTime {...args} />;
```

Comment:
You might need to mock the function needed to fetch the current time. As written now the "Future relative time" example is only in the future for another 6 days in the preview. https://deploy-preview-184--admirable-axolotl-a29e53.netlify.app/?path=/docs/base-relativetime--current-relative-time

Reply:
Anytime the preview is refreshed the times are reset and display correctly. I tested this by changing the future time to 5 minutes in the future and resetting it by refreshing the page.

Reply:
I tested this by changing the future relative time to 5 minutes in the future. Refreshing the page resets the current date. And therefore resets the future and past dates. However the 5 minutes in the future is never displayed again. It always defaults to 4 minutes in the future after the first refresh. I am going to test the behaviour with days but it seems like refreshing the page resolves this issue.

Reply:
Not sure if it is still a problem, but before the preview version would have the unintended time persist through refresh. Best I can tell the underlying component is updating the current time correctly but the future time is static in the preview.